models/surveys.go

```diff
@@ -846,6 +922,16 @@ func (api *API) GetClassifierTypeSelectorByID(w http.ResponseWriter, r *http.Req
 	surveyID := vars["surveyId"]
 	classifierTypeSelectorID := vars["classifierTypeSelectorId"]
 
+	if !IsValidUUID(surveyID) {
+		http.Error(w, "'"+surveyID+"' is not a valid UUID", http.StatusBadRequest)
+		return
+	}
+
+	if !IsValidUUID(classifierTypeSelectorID) {
+		http.Error(w, "'"+classifierTypeSelectorID+"' is not a valid UUID", http.StatusBadRequest)
```

Comment:
You're not saying in the error which one was wrong - just the value itself. Would be useful to know which param was wrong.

Could also streamline a little:
```go
for _, u := range []string{"classifierTypeSelectorId", "surveyId"} {
        val, ok := vars[u]
	if !ok {
		// ... client error empty mandatory param
	}
	
	if _, err := uuid.FromString(val); err != nil
		// ... client error not a uuid
	}
}

// Can now safely do:
surveyID := vars["surveyId"]
classifierTypeSelectorID := vars["classifierTypeSelectorId"]
```